homeassistant/components/cloud/http_api.py

```diff
@@ -815,5 +824,12 @@ def tts_info(
 ) -> None:
     """Fetch available tts info."""
     connection.send_result(
-        msg["id"], {"languages": [(lang, gender.value) for lang, gender in MAP_VOICE]}
+        msg["id"],
+        {
+            "languages": [
+                (language, voice)
```

Comment:
The languages container has now the following structure:

```py
[('af-ZA', 'AdriNeural'),
 ('af-ZA', 'WillemNeural'),
 ('am-ET', 'MekdesNeural'),
 ('am-ET', 'AmehaNeural'),
 ...
]
```

It used to have this structure:

```py
[('af-ZA', 'female'),
 ('af-ZA', 'male'),
 ('am-ET', 'female'),
 ('am-ET', 'male'),
 ...
]
```